### DISCUSSION TITLE
port option is not work in config.rb

### DISCUSSION THREAD

**Discussion by oeyoews** [gollum/gollum#1838] [Q&A]
![image](https://user-images.githubusercontent.com/72405338/180634638-f03566a8-14ba-4a26-876b-46d19fd231af.png)

if use `gollum --port 8082`, it's work,  but it isn't work in config.rb file, still is 4567

**Comment by benjaminwil** [comment]
Hello! I just spent some time looking into this. [After reading through `bin/gollum`](https://github.com/gollum/gollum/blob/master/bin/gollum) it became clear to me that `port` is not a `wiki_option` like the other setting in your screenshot, `ref` is.

So it looks like, currently, port cannot be controlled via a `config.rb` file. If there is some documentation that specifically mislead you about this, we should change it. And perhaps it would be useful to improve documentation around `config.rb` things generally.
